packages/functional-tests/tests/subscription-tests/subscription.spec.ts

```diff
@@ -53,3 +53,72 @@ test.describe('subscription test with cc and paypal', () => {
     expect(await relier.isPro()).toBe(true);
   });*/
 });
+
+test.describe('metrics - flow metrics query params', () => {
```

Comment:
Can we have a nested [`test.describe` here that indicates the severity of each test case](https://github.com/mozilla/fxa/blob/main/packages/functional-tests/README.md#group-by-severity)?

Any test where the user has opted out and we're still adding flow metrics should be higher severity than the others. I would think that would be a `'severity-2'`, and the others would be a `'severity-3`. Might be a good idea to confirm with Stef and Chelsea.